demo/DemoArticles/articles.json

```diff
@@ -0,0 +1,24 @@
+[ 
+  {"lang": "en", "title": "Barack Obama", "revision": 793008506},
```

Comment:
You'd get some nice typing at the source of creation. Instead of an anonymous object that you convert to a type, you just instantiate the type directly.